src/styles/_catalog.scss

```diff
@@ -1,6 +1,20 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-items: center;
+  grid-template-columns: repeat(4, minmax(200px, 1fr));
+  gap: 48px;
+
+  @media (max-width: 1024px) {
```

Comment:
Optional: you can use cycle for media just for convenience)

Reply:
Unfortunately, I could not use cycle, it doesn't work for some reason

Reply:
looks at others' cycles, if you made the same and it still doesn't work - good reason to visit QnA 😉